draft-ietf-pquip-pqc-engineers.md

```diff
@@ -263,12 +283,16 @@ Post-quantum cryptography or quantum-safe cryptography refers to cryptographic a
 
 When considering the security risks associated with the ability of a quantum computer to attack traditional cryptography, it is important to distinguish between the impact on symmetric algorithms and public-key ones. Dr. Peter Shor and Dr. Lov Grover developed two algorithms that changed the way the world thinks of security under the presence of a CRQC.
 
-## Symmetric cryptography {#symmetric}
+It is also worth some discussion of the term "quantum adversary". Quantum computers are, by their nature, hybrids of classical and quantum computational units; Shor's algorithm is, for example, largely composed of classical computation with one quantum computation step. In this way, the term "quantum adversary" should be thought of as "quantum _enhanced_ adversary"; ie they have access to the full range of classical and quantum computational techniques, as well as hybrid computational techniques.
```

Comment:
This goes too far.  Calling an entire quantum fast fourier transform "one quantum computation step" and implying the algorithm is "largely composed" of  classical operations despite the fact that they're just some VERY simple classical number theory is I think a bit misleading.  Perhaps just tossing "largely" fixes this.

Reply:
I propose to simplify the text as follows: 

It is also worth discussing the term 'quantum adversary.' Quantum computers are, by their nature, hybrids of classical and quantum computational units. For example, Shor's algorithm primarily consists of classical computations with one quantum computation step. Thus, the term 'quantum adversary' should be considered as 'quantum-enhanced adversary,' meaning they have access to both classical and quantum computational techniques.